nomad/nomad_jobs/traefik_job.hcl.j2

```diff
@@ -13,10 +13,13 @@ job "traefik" {
 
     network {
       port  "http" {
-         static = 80
+          static = 80
       }
       port  "admin" {
-         static = 8080
+          static = 8080
+      }
+      port  "deployments" {
+          static = 8081
```

Comment:
what does this port do?

Reply:
I was just trying some changes while testing. Removed it now.